.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: Build
```

Comment:
Do you have an idea on how we could setup the different stages of our pipeline? Looking at the documentation it seems like we could do either multiple workflows or multiple steps. Also jobs run in parallel, which could be interesting when running unit tests and integration tests for example.

Reply:
It's possible to have multiple .yml files in `.github/workflows` for multiple workflows.

Reply:
Here is a good reference to understand the [workflow syntax for github action](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions).